cmd/fetchkeys/flag.go

```diff
@@ -0,0 +1,21 @@
+package fetchkeys
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagSOPSKeysDir = "sops-keys-dir"
+)
+
+type flag struct {
+	SOPSKeysDir string
+}
+
+func (f *flag) Init(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&f.SOPSKeysDir, flagSOPSKeysDir, "", `Directory to fetch SOPS private keys into (optional)."`)
+}
+
+func (f *flag) Validate() error {
+	return nil
```

Comment:
Would it make sense to validate if the given path exists? If i understand correctly we [generate a tmp dir](https://github.com/giantswarm/konfigure/pull/121/files#diff-ec6e26b18fe7f3820909be13cc88df4ec2b7b37e7918f940020d3e6007ace63aR90-R98) if this is not set. Then at usages ([for example](https://github.com/giantswarm/konfigure/pull/121/files#diff-ec6e26b18fe7f3820909be13cc88df4ec2b7b37e7918f940020d3e6007ace63aR261)) we simply write the file at the given folder. If the folder does not exists or maybe not writable then the error would come up from those locations.

Reply:
Hmm, not a bad idea, not sure though if it is better to validate existence in the commands or move it down to `SOPSEnv`, `SOPSEnv` seems better so that we do not need to place it for every command